apps/aeutils/priv/extensions/get_peer_key.sh

```diff
@@ -1,9 +1,34 @@
 #!/bin/bash
 
+KEYSDIR=${1:-"$HOME/node/keys"}
+PASS=${2:-"top secret"}
+
 # get the peer pubkey
 CODE="{ok, PK} = aec_keys:peer_pubkey(),
       binary_to_list(aec_base58c:encode(peer_pubkey, PK))."
 
-relx_nodetool eval $CODE
+! relx_nodetool eval $CODE
+
+if [ $? -eq 1 ]; then
+    exit 0
+fi
+
+CODE="{ok, Bin} = file:read_file(filename:join(\"$KEYSDIR\", \"peer_key.pub\")),
```

Comment:
Not sure why this second code attempt is needed?

Reply:
If the epoch node is not running/not started we read the key directly from the file. Will be good on failed deployments.